scripts/superdesk/editor2/editor.ctrl.js

```diff
@@ -366,7 +366,8 @@ function SdTextEditorController(_, EMBED_PROVIDERS, $timeout, $element, editor,
                 }
                 return hash;
             }
-            return String(Math.abs(hashCode(block.body))) + String(vm.getBlockPosition(block));
```

Comment:
By doing this, it prevents the editor to support two identical blocks (same content or same embed link)